src/praisonai-agents/tests/unit/memory/test_dakera_adapter.py

```
@@ -0,0 +1,322 @@
+import sys
+import types
+
+import pytest
+
+
+# ---------------------------------------------------------------------------
+# Fake dakera SDK
+# ---------------------------------------------------------------------------
+
+class _FakeRecalledMemory:
+    def __init__(self, id, content, memory_type, importance=0.5, score=0.9,
```

Ruff (0.15.20)

[error] Function argument `id` is shadowing a Python builtin

(A002)